### ISSUE TITLE
False alarm with commas in JSX

### ISSUE THREAD

**Issue by ustun** [eslint/eslint#1916]
Putting a comma in a JSX expression causes a false alarm.

```
var X = React.createClass({

    render: function () {
        var hello = "Hello";
        var world = "World";
        return <span>{hello}, {world}</span>;
    }

});
```

```
{
  "ecmaFeatures": {
    "jsx": true
  }
}
```

```
$ eslint index.js

  ...
  6:28  error  A space is required after ','   comma-spacing
```

See https://github.com/babel/babel-eslint/issues/18 for the original report. It was found out the issue was upstream.

**Comment by nzakas** [comment]
Confirmed. We should not flag commas in JSX elements.

**Comment by btmills** [comment]
Working on this.

**Comment by ustun** [comment]
Thank you @btmills !
